backend/src/test/java/de/tobias/intestinalinspector/service/DateServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package de.tobias.intestinalinspector.service;
+
+import de.tobias.intestinalinspector.SpringBootTests;
+import de.tobias.intestinalinspector.api.FoodDto;
+import de.tobias.intestinalinspector.api.PainDto;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DateServiceTest extends SpringBootTests {
+
+    @Autowired
+    DateService dateService;
+
+
+    @Test
+    public void testSortFood(){
+        //GIVEN
```

Comment:
This is a unit test, no need to start the spring context.

```suggestion
class DateServiceTest {

    DateService dateService = new DataService(); //or init in each test


    @Test
    public void testSortFood(){
        //GIVEN
```